superset/security/manager.py

```diff
@@ -81,7 +81,11 @@ def __init__(self, **kwargs):
 PermissionModelView.list_widget = SupersetSecurityListWidget
 
 # Limiting routes on FAB model views
-UserModelView.include_route_methods = RouteMethod.CRUD_SET | {"userinfo"}
+UserModelView.include_route_methods = RouteMethod.CRUD_SET | {
+    RouteMethod.ACTION,
+    RouteMethod.ACTION_POST,
+    "userinfo",
```

Comment:
The `userinfo` looks lonely here with the constants. Wondering if we should collect these somewhere, too.

Reply:
We can of course, did a quick search and found a bunch "lonely" routes. Like `download_dashboards`, `bulk_delete`, `table_metadata`. Would say that the idea here is that non repeatable ones, get referenced in place.

Reply:
Perhaps not something that needs to be addressed now (not really sure which is better). LGTM